storage/storage.cpp

```diff
@@ -1307,6 +1308,20 @@ void Storage::GetNodeAttrs(TCountryId const & countryId, NodeAttrs & nodeAttrs)
   nodeAttrs.m_downloadingProgress =
       CalculateProgress(downloadingMwm, descendants, downloadingMwmProgress, setQueue);
 
+  // Local mwm information.
+  node->ForEachInSubtree([this, &nodeAttrs](TCountriesContainer const & d)
+  {
+    auto const localeMapIt = m_localFiles.find(d.Value().Name());
```

Comment:
What about to fix name localeMapIt -> localMapIt. It is not about locale, it is about local files, right?